pkg/monitortests/node/nodestateanalyzer/node.go

```diff
@@ -61,6 +62,10 @@ func intervalsFromEvents_NodeChanges(events monitorapi.Intervals, _ monitorapi.R
 			if event.Source == monitorapi.SourceNodeMonitor {
 				nodeStateTracker.OpenInterval(nodeLocator, notReadyState, event.From)
 			}
+		case "Unknown":
+			if event.Source == monitorapi.SourceNodeMonitor {
+				nodeStateTracker.OpenInterval(nodeLocator, unknownState, event.From)
+			}
```

Comment:
at this point in the code, the ready/notready is accurate for our purposes.  I don't think we need the new unknown state here.